proteuslib/property_models/tests/property_test_harness.py

```diff
@@ -304,6 +304,16 @@ def test_badly_scaled(self, frame_stateblock):
             raise PropertyValueError(
                 "The following variable(s) are poorly scaled: {lst}".format(lst=lst))
 
+    @pytest.mark.component
+    def test_calculate_state(self, frame_stateblock):
+        m = frame_stateblock
+
+        # check if calculate_state runs and solves rather than how its used
```

Comment:
Can you add a test that demonstrates how supplying volumetric feed flow rate and mass concentration to `calculate_state` would lead to some expected set of results?

Reply:
Yes, I haven't finished the PR yet. I'm planning on adding an individual test class in the property test harness for calculate_state cases. I'm also going to use it in the RO with energy recovery flowsheet.